src/pages/objkt-display/index.js

```diff
@@ -78,10 +81,14 @@ export const ObjktDisplay = () => {
     )
 
     objkt.listings = listings
-    objkt.ban = getWalletBlockList()
+
+    if (nsfwList.includes(objkt.id)) {
+      objkt.content_rating = METADATA_CONTENT_RATING_MATURE
+    }
+
     await context.setAccount()
 
-    if (objkt.ban.includes(objkt.creator.address)) {
+    if (walletBlockList.includes(objkt.creator.address)) {
```

Comment:
wondering if it wouldn't be better to make `walletBlockList` a hashmap to make these lookups less expensive.

Reply:
Like {address: restrictedStatus} ? I can look into that 👁️